diffusion/train.py

```diff
@@ -16,6 +17,37 @@
 from composer.loggers import LoggerDestination
 from composer.utils import dist, reproducibility
 from omegaconf import DictConfig, OmegaConf
+from torch.optim import Optimizer
+
+from diffusion.models.autoencoder import ComposerAutoEncoder, ComposerDiffusersAutoEncoder
+
+
+def make_autoencoder_optimizer(config: DictConfig, model: ComposerModel) -> Optimizer:
+    """Configures the optimizer for use with an autoencoder + discriminator loss."""
+    print('Configuring opimizer for autoencoder+discriminator')
```

Comment:
Wait, shouldn't this use a logger instead of a print statement?

Reply:
Probably, yeah. But the rest of `train.py` uses print statements so I'm going to hold off on fixing this for a PR that fixes all of them.